### ISSUE TITLE
AS1/2 No logging + For..in..return decompilation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/asdec#40]
```
What steps will reproduce the problem?
1. Try to decompile obfuscated or damaged SWF file.

What is the expected output? What do you see instead?
The program hangs but in version 1.3.0 and lower it used to print stack trace 
into log.txt file. Now it stays empty after crash.

What version of the product are you using? On what operating system?
1.4.0 - 1.4.0u1
Windows 7 64 bit

Could you provide the SWF file you have problem with?
If the answer is yes, then please attach it here or send me it via email.
Could you at least attach PCode source?
You can check it with the file from issue #39 - try to see source code for 
DoInitAction 45. And I suppose its another issue by itself.
I have attached log file from 1.3.0 version.

Do you have the original source code which produced the wrong
decompilation? If yes, then please attach it.

Please provide any additional information below.


```

Original issue reported on code.google.com by `plisnich...@gmail.com` on 5 Mar 2013 at 9:46

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/asdec/issue-40/comment-0/log.txt)

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 11 Mar 2013 at 6:55
- Changed title: **AS1/2 No logging + For..in..return decompilation**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jindra.p...@gmail.com` on 11 Mar 2013 at 9:07
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hi,
it is fixed in version 1.4.2, try it.
```

Original comment by `jindra.p...@gmail.com` on 24 Mar 2013 at 11:51
- Changed state: **Fixed**
